libraries/app/include/graphene/app/database_api.hpp

```diff
@@ -685,7 +692,7 @@ class database_api
        *  For each operation calculate the required fee in the specified asset type.  If the asset type does
        *  not have a valid core_exchange_rate
```

Comment:
While you're on it, perhaps complete this sentence?

Reply:
I don't see how the core exchange for an asset can be invalid. Even if so, i can't see any special check for it in the called functions to get the output. I removed the incomplete sentence but open to suggestions if i am not seeing something.